apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -68,12 +190,38 @@
 
   li {
     display: inline-block;
-    width: 100px;
-    margin: 0 10px;
+    width: 49%;
+    height: 0;
+    margin: 0;
     cursor: pointer;
+    padding-bottom: 50%;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)